jobs.groovy

```diff
@@ -473,7 +473,13 @@ def list_jobs(dashProject, dashBranchName) {
                                  module: module, values: job_values,
                                  full_name: full_name])
                 }
-            } else {
+            } else if (job_values.module) {
+                full_name = full_job_name(
+                    dashProject, dashBranchName, job_name)
+                results.add([type: job_type, name: job_name,
+                             module: job_values.module, values: job_values,
+                             full_name: full_name])
```

Comment:
Not a huge fan of duplicating this type information across languages, but can't see an alternative.